labs/jack-in/src/config.rs

```diff
@@ -0,0 +1,81 @@
+use std::path::PathBuf;
+
+use matrix_sdk::SlidingSyncMode;
+use structopt::{clap::arg_enum, StructOpt};
```

Comment:
Doesn't have to be this PR, but I think we should switch to just clap (v4), which has all of structopt's functionality.

Reply:
noted: https://github.com/matrix-org/matrix-rust-sdk/issues/1274